internal/controller/datadogagent/feature/apm/const.go

```diff
@@ -13,6 +13,10 @@ import (
 
 const (
 	apmRBACPrefix = "apm"
+
+	apmHostPortName          = "traceport"
+	apmSocketVolumeName      = "apmsocket"
+	apmSocketVolumeLocalPath = "/var/run/datadog"
```

Comment:
Should this be combined with dsd like we do with the hostpath `DogstatsdAPMSocketHostPath`? Or would it be better to separate `DogstatsdAPMSocketHostPath` into dsd and apm specific paths?

Reply:
i'm not sure, but i guess we can decide and stick with one approach. it certainly seems there is some deduplication we can do for this particular constant